qt/qt_common/map_widget.cpp

```diff
@@ -395,9 +396,10 @@ void MapWidget::initializeGL()
         // TODO: Separate apiOpenGL3 from apiOpenGLES3, and use that for the currend shader code.
         m_apiOpenGLES3 = true;
       }
+
+      fmt.setRenderableType(QSurfaceFormat::OpenGLES);
```

Comment:
1. A comment should be added.
2. It may not work on Mac and on Windows, should be tested first.

Reply:
So far, it doesn't seem to work even on Linux, meaning the context doesn't actually switch. I don't know why yet:
https://github.com/flathub/app.organicmaps.desktop/issues/50#issuecomment-1633167565